xonsh/prompt/base.py

```diff
@@ -338,9 +338,10 @@ def load_initial(self):
         self.update(
             dict(
                 user=xp.os_environ.get(
-                    "USERNAME" if xp.ON_WINDOWS else "USER", "<user>"
+                    "USERNAME" if xp.ON_WINDOWS else "USER",
+                    "root" if xt.is_superuser() else "<user>",
```

Comment:
Is this needed? 

What's the situation where the user is a superuser but the username isn't `root`?